docs/data/scheduler/overview/DayGridPrimitives.module.css

```diff
@@ -0,0 +1,73 @@
+.Container {
+  --border-color: oklch(87.1% 0.006 286.286);
+  --event-work-bg-color: oklch(82.8% 0.111 230.318);
+  --event-personal-bg-color: oklch(81% 0.117 11.638);
+}
+
+:global(.mode-dark) .Container {
+  --border-color: oklch(44.2% 0.017 285.786);
+  --event-work-bg-color: oklch(50% 0.134 242.749);
+  --event-personal-bg-color: oklch(45.5% 0.188 13.697);
+}
+
+.Root {
+  border-collapse: collapse;
+  border-radius: 4px;
+  display: flex;
+  flex-direction: column;
+}
+
+.Row {
+  border-bottom: 1px solid var(--border-color);
+  display: flex;
+
+  &:first-child {
+    border-top: 1px solid var(--border-color);
+  }
+}
+
+.Cell {
+  display: flex;
+  flex-direction: column;
+  height: 120px;
+  width: 120px;
```

Comment:
There’s a little CSS trick you can use here 🪄 
Instead of setting a fixed width and height, you can use `aspect-ratio: 1 / 1`, it keeps the cells square but makes the layout more flexible on slightly smaller screens.
You need to add also `display: grid` with `grid-template-columns: repeat(6, 1fr)` (7 after the bug fix) in the `Row` instead of flex.
Also, removing the double border it should scale a lot better overall 🙌 
```suggestion
aspect-ratio: 1 / 1;
```

Reply:
Before:
https://github.com/user-attachments/assets/4bd948f6-3ff2-4b8b-b37a-c762818e09c4

After:
https://github.com/user-attachments/assets/ad367d63-778d-4998-a00a-29eacddbb415

Reply:
![image](https://github.com/user-attachments/assets/9e66740d-bc86-4516-87e7-922d9f2d0f4d)

The issue is that if you have 0 event the the cells are super tiny :laughing:

Reply:
Oh sorry! I forgot to add one more thing, you need in the main Container "width: 100%" so the component will cover all the available space, that makes the component flexible and gives all the control to the user 🙌
<img width="1412" alt="Screenshot 2025-05-28 at 15 33 11" src="https://github.com/user-attachments/assets/4550e8af-b159-4fec-bc9b-ef4c1fd7a1da" />

Reply:
Good catch by the way! 👏